packages/twenty-server/src/engine/core-modules/admin-panel/dtos/admin-panel-indicator-health-status.input.ts

```diff
@@ -4,7 +4,7 @@ export enum AdminPanelIndicatorHealthStatusInputEnum {
   DATABASE = 'database',
   REDIS = 'redis',
   WORKER = 'worker',
-  MESSAGE_SYNC = 'messageSync',
+  ACCOUNT_SYNC = 'accountSync',
```

Comment:
I think we used "connectedAccount" in the codebase so I probably would have stuck to that "Connected Account Status", etc